velox/exec/OutputBuffer.h

```diff
@@ -74,6 +74,41 @@ class ArbitraryBuffer {
   std::deque<std::shared_ptr<SerializedPage>> pages_;
 };
 
+struct BufferInfo {
+  void recordAddPage(const SerializedPage& data) {
+    VELOX_CHECK(data.rows() >= 0, "SerializedPage's row size is not inited");
```

Comment:
VELOX_CHECK_GE for better error messages

Reply:
done.